product-service/handlers/getProductById/getProductById.ts

```diff
@@ -0,0 +1,40 @@
+import { APIGatewayProxyHandler } from "aws-lambda";
+import sourceMapSupport from 'source-map-support'
+sourceMapSupport.install();
+import * as api from '../../api';
+import {corsHeaders} from '../constants';
+
+export const getProductById: APIGatewayProxyHandler = async (event) => {
+    const headers = {
```

Comment:
I think you can reuse already exisiting cors headers in `../constants`

Reply:
Fixed